### DISCUSSION TITLE
Getting into and our to vehicles

### DISCUSSION THREAD

**Discussion by lystraeudaimon** [fvtt-fria-ligan/twilight2000-foundry-vtt#48] [Q&A]
So you can add crew to vehicles, which is AWESEOME, but I've run into a problem where actors are getting into and out of the vehicles during the encounter. How is the best way to handle this? Right now I am dragging tokens out of the actors folders and putting them onto the map but it doesn't seem to update the token's. Just looking for the best practice here.

**Comment by Stefouch** [comment]
Yes, I would do like you said.
But I admit that just draggin' the crew out of the vehicle should remove it from the vehicle and create a token on the map. It would be a good suggestion.
